pkg/api/bzz.go

```diff
@@ -118,18 +118,18 @@ func (s *server) bzzDownloadHandler(w http.ResponseWriter, r *http.Request) {
 
 	manifestEntryAddress := me.Reference()
 
-	var additionalHeaders http.Header
```

Comment:
@mortelli Please note that `http.Header` is actually alias for type `map[string][]string`, which is why plural was used.

Reply:
yes, thanks. i think both singular and plural are arguably acceptable here.

in any case, i reverted this change _only_ for `bzz.go` since i want this part of the code to be consistent with the `downloadHandler` signature.